source/package.template.html.erb

```diff
@@ -57,7 +57,7 @@
                       style="min-width: 225px"
                       id="version"
                       onchange="this.options[this.selectedIndex].value && (window.location = this.options[this.selectedIndex].value);">
-                <% for local_version, package_version in package.versions.sort.reverse %>
+                <% for local_version, package_version in package.versions.sort_by { |v| Gem::Version.new(v[0]) }.reverse %>
```

Comment:
Thanks for explaining that @b-per!

Yes, I see now: `package.versions` is a hash where the [key is the version number](https://github.com/dbt-labs/hub.getdbt.com/blob/89b1f00aa60208816eb82f5d7dab3d4224e672b5/config.rb#L32-L43) (without any "v" prefix), and the value is of type `Middleman::Util::EnhancedHash` (which doesn't matter for the purposes of sorting).

The following suggestion isn't any more correct than what you have currently, but I think it is slightly easier to read and aligns with Ruby style conventions (I think!):
```suggestion
                <% for local_version, _package_version in package.versions.sort_by { |key, _val| Gem::Version.new(key) }.reverse %>
```

Since both `_package_version` and `_val` are unused, they contain a leading underscore to utilize the [Ruby convention for unused variables](https://github.com/rubocop/ruby-style-guide#unused-variables-prefix).

What do you think?

Reply:
I'm going to commit this suggestion and merge it since we can always change it back to the original `v[0]` syntax if we don't like the suggested syntax.